Makefile

```diff
@@ -20,6 +20,10 @@ GO_BUILD_PACKAGES :=$(strip \
 	./cmd/... \
 	$(wildcard ./tools/*) \
 )
+GO_TEST_PACKAGES :=$(strip \
+	./cmd/... \
```

Comment:
missing `./pkg/...`

Reply:
Yeah, that was meant to be `./...` actually, like in `GO_PACKAGES ?=./...`